evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -743,7 +743,16 @@ impl<'a, 'b, B: Backend, P: PrecompileSet> Handler for CheatcodeStackExecutor<'a
         } else if code_address == *CONSOLE_ADDRESS {
             self.console_log(input)
         } else {
-            self.handler.call(code_address, transfer, input, target_gas, is_static, context)
+            self.call_inner(
```

Comment:
Lifted from Sputnik executor's `call` function [here](https://github.com/rust-blockchain/evm/blob/953f6cb16a07458d139e8dec5bafdc6cff4f6c4b/src/executor/stack/executor.rs#L1096). 

This ignores the `emit_exit!` used in the call function they have when `tracing` is enabled, but I noticed that we're not using any `event!`s in `call_inner` or `create_inner` either, so it seemed ok. Lemme know if that's not a reasonable assumption